replay_build_scripts/upload_build_artifacts.mjs

```diff
@@ -155,9 +157,9 @@ function prepareWindowsBinaries(buildId) {
   return [buildArchive];
 }
 
-function prepareMacOSBinaries(buildId) {
-  const dmgArchive = `${buildId}.dmg`;
-  const outdir = path.join("out", "Release");
+function prepareMacOSBinaries(buildId, buildArm) {
```

Comment:
Adding `buildArm` as a argument here will shadow the module variable causing it to always be `undefined`, right?

Reply:
Ugh. I miss TypeScript!

Reply:
(Thanks!)